server/game/cards/02_SHD/upgrades/TopTarget.ts

```diff
@@ -0,0 +1,28 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { UpgradeCard } from '../../../core/card/UpgradeCard';
+import { CardType, KeywordName, RelativePlayer } from '../../../core/Constants';
+
+export default class TopTarget extends UpgradeCard {
+    protected override getImplementationId() {
+        return {
+            id: '4282425335',
+            internalName: 'top-target',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addGainKeywordTargetingAttached({
+            keyword: KeywordName.Bounty,
+            ability: {
+                title: 'Heal 4 damage from a unit or base. If the Bounty unit is unique, heal 6 damage instead.',
```

Comment:
Suggestion: This can also heal a unit so we need logic allow targeting of units.